config.toml

```diff
@@ -81,6 +81,11 @@ algolia_docsearch = false
 prism_syntax_highlighting = true
 project_home = "https://openservicemesh.io/"
 
+# Version params
+osm_version = "v0.11"
+osm_full_version = "v0.11.0"
+envoy_version = "v1.17.2"
```

Comment:
Would it make sense to add an `osm_branch` variable so we could also point to `main` in addition to release branches?

Reply:
```suggestion
# Version params
osm_version = "v0.11"
osm_full_version = "v0.11.0"
envoy_version = "v1.19.1"
```

Reply:
Is this what you are suggesting?

```suggestion
# Version params
osm_branch = "release-v0.11"
osm_version = "v0.11.0"
envoy_version = "v1.19.1"
```

For the main branch this could look like, 
```toml
osm_branch = "main"
osm_version = "v0.11.0"
envoy_version = "v1.19.1"
```

Reply:
Yeah that looks good. I'm not sure if you'd still need the vMAJOR.MINOR elsewhere outside the context of the release branch name though.

Reply:
As far as I know, there were no uses of vMAJOR.MINOR outside the context of the release branch name. This can be added later if needed.